Formula/solana.rb

```diff
@@ -21,10 +21,12 @@ class Solana < Formula
 
   depends_on "protobuf" => :build
   depends_on "rust" => :build
+  depends_on "rustfmt" => :build
```

Comment:
Why is this needed? Shouldn't the source be formatted correctly already?

Reply:
@SMillerDev The code base uses [prost](https://github.com/tokio-rs/prost), a Protobuf generator for Rust. Rust seems to format all generated code before compiling. I think the point is to allow deterministic line numbers in backtraces for auto-generated files.

Reply:
Can you ask upstream for an option to skip this?